bin/dist.sh

```
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -ex
+
+PLUGIN_NAME='vk-google-job-posting-manager'
+PLUGIN_DIR=$(cd $(dirname $(dirname $0)); pwd)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)